### ISSUE TITLE
Can you tell me about the GPUs you use?

### ISSUE THREAD

**Issue by zks1122** [JiahuaDong/HFC#3]
Can you tell me about the GPUs you use, what GPUs you use, and how many GPUs you use? Thank you very much.

**Comment by WenqiLiang** [comment]
Of course, we conduct our experiments on a single Tesla A100 80G with a GPU memory load of 22G.
